torch/_tensor_docs.py

```diff
@@ -6511,55 +6511,3 @@ def callable(a, b) -> number
 
 """,
 )
-
-add_docstr_all(
```

Comment:
You want to keep an entry for this method.
It can be simply `See torch.nested.to_padded_tensor()` (like many others in this file that point to torch.)